treeherder/perf/management/commands/backfill_perf_jobs.py

```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from treeherder.perf.backfill_tool import BackfillTool
+from treeherder.services.taskcluster import DEFAULT_ROOT_URL as root_url
+from treeherder.services.taskcluster import TaskclusterModel
+
+
+class Command(BaseCommand):
+    help = "Backfill missing performance jobs"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'job',
+            action='store',
+            type=str,
+            help="Performance job to backfill from",
+            metavar='JOB_ID',
+        )
+
+    def handle(self, *args, **options):
+        job_id = options['job']
+        client_id = settings.PERF_SHERIFF_BOT_CLIENT_ID
+        access_token = settings.PERF_SHERIFF_BOT_ACCESS_TOKEN
+
+        taskcluster_model = TaskclusterModel(root_url, client_id, access_token)
+        backfill_tool = BackfillTool(taskcluster_model)
+
+        task_id = backfill_tool.backfill_job(job_id)
```

Comment:
this handles one job at a time?  What if we already backfilled (if a job has multiple tests/datapoints that have regressed), what is to say we won't backfill the job many times?

Reply:
The sole purpose of this module is to act as a smoke test. We have absolutely no intention to use this on production or even on staging.
It only allowed me to entirely check the functionality of the backfill tool on my local dev environment.

@jmaher would a top module comment explaining this help?

Reply:
yes, please add a comment to this so it is clear.